guide/english/javascript/es6/arrow-functions/index.md

```diff
@@ -4,7 +4,7 @@ title: Arrow Functions
 
 ## Arrow functions
 
-ES6 has introduced a new syntax that allows to declare functions.
+ES6 has introduced a new syntax for function declarations.
```

Comment:
I'd say this change is a bit confusing and technically wrong.
Function declaration in JS means declaring function like:
```
function someFunc(){ }
```
Another way would be function expression:
```
const x = function(){ }
```
The arrow functions are always function expression and not *declaration*:
```
const x = () => {};
```

Reply:
@ezioda004 Good point!  I did not catch that.

Reply:
What do you suggest for a verbiage change if any?

Reply:
I'm happy to change it back. It seemed like it was grammatically incorrect to me. That's the only reason I changed it.

Reply:
What about the following?
`ES6 has introduced a new syntax for defining a function.`

Reply:
I'll change it to that. I'll make the changes once we hear back about the `console.log()`

Reply:
@RandellDawson I'd prefer *defining* instead of *declaring*, so your suggestion seems fine to me.

Reply:
@RandellDawson 

I made those changes. I added the log back into to the seed. I hope this was correct.